docs/pages/guides/devtools.mdx

```diff
@@ -0,0 +1,177 @@
+      type="video/mp4"
+    />
+  </video>
+</Figure>
+
+### With an example from our gallery
+
+All of [our examples](/examples) hosted on Liveblocks.io are available for you to run locally
+from the
+[examples directory](https://github.com/liveblocks/liveblocks/tree/main/examples)
+in the Liveblocks repo. For testing and reviewing how to use our DevTools, we
+will use the
+[Advanced Collaborative Spreadsheet](https://liveblocks.io/examples/collaborative-spreadsheet-advanced/nextjs).
+
+To clone the Git repository and run the example, run the following commands:
+
+```bash
+git clone https://github.com/liveblocks/liveblocks.git
+```
+
+```bash
+cd liveblocks/examples/nextjs-collaborative-spreadsheet-advanced
+```
+
+```bash
+npm install
+```
+
+Then, create the `.env.local` file with your `LIVEBLOCKS_SECRET_KEY`. Obtain
+your secret key from [here](https://liveblocks.io/dashboard/apikeys).
+
+```bash file=".env.local"
+LIVEBLOCKS_SECRET_KEY="sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx"
+```
+
+Open your browser and open up the developer tools window. Find the newly
+available “Liveblocks” panel there.
+
+<Figure highlight={false}>
+  <Image
+    src="/images/docs/devtools/spreadsheet-example.png"
+    alt="Liveblocks DevTools panel"
+    layout="responsive"
+    width={730}
+    height={504}
+    quality={90}
+  />
+</Figure>
+
+Now you can run our example by running `npm run dev` and navigating to
+`http://localhost:3000`, and you should see the values from the spreadsheet
+examples populating storage.
+
+## Features and tips
+
+### Highlighting
+
+When a value in storage changes, the DevTools will highlight the value to
+indicate that it has changed.
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source
+      src="/images/docs/devtools/highlight-changes.mp4"
+      type="video/mp4"
+    />
+  </video>
+</Figure>
+
+### Search
+
+Storage values are searchable by key and regex.
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source src="/images/docs/devtools/search-regex.mp4" type="video/mp4" />
+  </video>
+</Figure>
+
+### Hover
+
+Hovering over a value in storage or presence makes an eye icon appear. If
+clicked, it displays the value of that key.
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source src="/images/docs/devtools/expand-tooltip.mp4" type="video/mp4" />
+  </video>
+</Figure>
+
+### Expand and collapse
+
+- You can expand and collapse objects with one level of depth by using the arrow
+  keys or spacebar
+- You can quickly expand and collapse nested objects within storage by holding
+  the `⌥` key (option on Mac) and clicking on the object
+
+<Banner title="Initialization">
+
+If you open DevTools after a Liveblocks client has already been initialized, you
+may see an empty page. This is because DevTools is only able to retrieve the
+current state of storage and presence. To fix this, you can refresh the page or
+open DevTools before initializing the Liveblocks client.
```

Comment:
```suggestion
If you open the DevTools after a Liveblocks client has already been initialized, you
may see an empty page. This is because our DevTools is only able to retrieve the
current state of storage and presence. To fix this, you can refresh the page or
open the DevTools before initializing the Liveblocks client.
```